extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/rule/Authorization.scala

```diff
@@ -41,6 +48,7 @@ abstract class Authorization(spark: SparkSession) extends Rule[LogicalPlan] {
 object Authorization {
 
   val KYUUBI_AUTHZ_TAG = TreeNodeTag[Unit]("__KYUUBI_AUTHZ_TAG")
+  var EXPLAIN_COMMAND_EXECUTION_ID: Option[String] = None
```

Comment:
this works?

Reply:
> this works?

Yea

Reply:
> this works?

Maybe need change for multi session. Let me change it to ThreadLocal

Reply:
> > this works?
> 
> Maybe need change for multi session. Let me change it to ThreadLocal

Updated